### PR TITLE
Add functionality to draw pieces to the board

### PR DESCRIPTION
**Pull request by smlovin2**
- Adds a new helper method, draw_piece, to helpers/games_helper.rb
- Adds white? and black? methods to piece.rb
- Adds chess_font_representation method to all the different piece subclasses(ex. pawn.rb)
- Adds chess_casesregular font-family
- Updates games/show.html.erb to use the draw_piece method in order to print pieces out on the board
- Updates games.scss to use the chess font and try and size and align the pieces so they are in the middle of the space

**Notes and Questions**
- I couldn't get the view tests working :count thing in include so right now it only looks for one of each piece. I'm not sure how to look for multiples of something and couldn't really find anything helpful online yet. So might need to either just change the names of the test and only test for one each or if anyone knows of a way to test for multiples I would love some help with that :smile: 
- I wasn't able to really check manually if the pieces were drawn because there were some issues with the games index page and creating a new game. There seems to be a syntax error in games/index.html.erb on line 19. I couldn't figure out what it was, so I just removed that line and then the index page worked but when I tried to make a new game by clicking the 'New Game' link I got another error because that link creates a form that then looks for the name of the game but since the name column was removed there is no name so it fails. So I could possiblly remove that line or something as well and then it might work but for right now I didn't want to mess with it too much. It looks like that code might need some cleaning up though.
- I forgot to run rubocop first so I'll probably need to make some changes for that as well.

### CODE REVIEW

File: spec/helpers/games_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+# Specs in this file have access to a helper object that includes
+# the GamesHelper. For example:
+#
+# describe GamesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```
Comment (mscoutermarsh) [inline spec/helpers/games_helper_spec.rb]: you can delete any commented code

File: spec/helpers/games_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+RSpec.describe GamesHelper, type: :helper do
+  describe '#draw_piece' do
+    it 'returns nil when there is no piece at the given location' do
+      game = FactoryGirl.create(:game)
```
Comment (mscoutermarsh) [inline spec/helpers/games_helper_spec.rb]: Tests look great.

One thing you can do to DRY this up a bit.

After line 14, add `let(:game) { FactoryGirl.create(:game) }`

Then you can remove `game = FactoryGirl.create(:game)` everywhere.

**Comment by mscoutermarsh** [comment]
Looks great. I'm not sure about the tests either. Would be good to figure out the count. Must be a way somehow. If you need help figuring it out, let me know.

Merge it whenever you're ready :)

**Comment by smlovin2** [comment]
@mscoutermarsh Found a helpful gem yesterday during mentoring session called rspec-html-matchers that allowed me to do the thing I was hoping to do. I updated the view tests and now we are actually looking for the specific amount of pieces. Am I still good to merge it or did you want to look at the tests?

**Comment by mscoutermarsh** [comment]
:+1:

**Pull request merged**